source/juiceness/pop_score/pop_label.gd

```diff
@@ -1,8 +1,14 @@
 extends Node2D
-
+# Cannot convert the provided value to the export type. That's why I commented them out.
+"""
 export (float) var final_scale = Vector2(1.5, 1.5)
 export (float) var float_length = 100
 export (float) var duration = 0.25
```

Comment:
I think that the problem here was because `export (float) var final_scale = Vector2(1.5, 1.5)` was supposed to be `export(Vector2) var final_scale = Vector2(1.5, 1.5)`, I will fix that

Reply:
Done at https://github.com/pigdevstudio/platform_template/pull/48/commits/fa6d23291c657a134708058f4f4e399d716d289b :+1: